### ISSUE TITLE
Which Jython version works with Python 3.9

### ISSUE THREAD

**Issue by Emily-Jiang** [jython/jython#312]
I need to use Python and Jython together. The Python version I need is 3.9. Which Jython release works with that Python 3.9?

**Comment by Emily-Jiang** [comment]
I found out the answer is no. Does anyone have a timeline on when the next Jython 3.0 will be released?

**Comment by jeff5** [comment]
No, no-one knows that, sorry. It will be a 3.x (where x >=10) as we name versions after the Python they support.

**Comment by HolgerHees** [comment]
I would like to vote for this too, because I depend on it for my homeautomation.

https://www.openhab.org/addons/automation/jythonscripting/

Currently I have more then 10.000 lines of python code running inside openhab. But the openhab team marks jython as deprecated, because they "feel" that the development stops and they fear that at a certain point new Java versions will no longer be supported and they will not be able to fix this.

for me personally it would be i big loose, because I love it to write automation rules in python which runs on top of openhab.

https://github.com/HolgerHees/openhab-config/tree/master/conf/automation/jsr223

maybe my comment is a motivation for someone to clarify this situation with the openhab team to increase their motivation to still keep jython support.

**Comment by Stewori** [comment]
It would be important to note that Python 3 support and maintaining compatibility with new Java versions are two different things, the former being the far bigger challenge. I see no reason for deprecating Jython since I am somewhat confident that we are keeping up with Java. That is why there is still a 2.7.4 release upcoming and releases are going to continue. I vaguely remember an email discussion, maybe on the dev list, where we decided that **P**ython 2 EOL cannot mean **J**ython 2 EOL as long as no Jython 3 exist to take over. So, since Jython 3 may still be years away, the same applies to Jython 2 EOL. However we cannot speak for the ecosystem and PyPI. It might get harder to obtain Python 2 libs, which are beyond our control.

**Comment by shayneoneill** [comment]
IMHO the issue really isn't Java compat (Java does a pretty decent job of backwards compatibility) its that cpython2.x has largely been EOLed and its unlikely the majority of libs that still with with 2.x are going to recieve updates. 

Still, thats how Open source rolls. Aint much point complaining if assistance isn't offered (I'd offer, but I wouldnt know how to write good Java if my life depended on it lol)

**Comment by HolgerHees** [comment]
maybe someone who has some insights, can influence the discussion in the openhab community about this topic.

https://community.openhab.org/t/jython-is-not-deprecated/154187

**Comment by jeff5** [comment]
Unfortunately, we're maintaining _an implementation of_ a deprecated version of Python. I think that would be a fair statement. Which I feel bad about, but seems best for users. Jython can't really be deprecated (from this side) without offering a replacement. That's still the aim.

I continue to be surprised, with each new Java LTS version, and each update to the dependencies, that not too much breaks. Look out for 2.7.4b1.
